.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
+
```

Comment:
this file has two empty lines at the end, it is just needed one empty line.

Reply:
fixed it